tests/unittests/qml/tst_WebProcessMonitor.qml

```diff
@@ -55,16 +53,19 @@ WebProcessMonitor {
             monitor.webview = webviewMock
             compare(monitor.killedRetries, 0)
 
-            webviewMock.webProcessStatus = Oxide.WebView.WebProcessKilled
+            webviewMock.renderProcessTerminated(WebEngineView.KilledTerminationStatus)
             verify(monitor.killed)
             verify(!monitor.crashed)
-            tryCompare(monitor, "killedRetries", 1)
-            tryCompare(webviewMock, "reloadCalled", 1)
+            wait(200)
+            compare(monitor.killedRetries, 1)
+            compare(webviewMock.reloadCalled, 1)
```

Comment:
Why was this moved from tryCompare to a wait followed by a compare?

Reply:
I think it was because the rest of the test_killed() method uses compare().
It was a long time ago, therefore I don't exactly remember, but I think there was an error when running it.

Reply:
ok it seems to build using the old way, too -> restoring it for now